src/SourceLocator/Type/DirectorySourceLocator.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: HP
+ * Date: 09/24/2016
+ * Time: 5:29 AM
+ */
+
+namespace BetterReflection\SourceLocator\Type;
+
```

Comment:
Remove this line (one empty line between `namespace` and the first `use` statement